src/Validator.php

```diff
@@ -13,73 +13,66 @@
  * Use https://github.com/vlucas/valitron under the hood.
  *
  * $v = new \Atk4\Validate\Validator($model);
+ *
+ * https://github.com/vlucas/valitron/blob/v1.4.11/src/Valitron/Validator.php#L1251
+ *
+ * @phpstan-type ValidatorCallback \Closure(string, mixed, list<mixed>, list<mixed>): bool
```

Comment:
```suggestion
 * @phpstan-type ValidatorCallback \Closure(string, mixed, list<mixed>, list<mixed>): bool https://github.com/vlucas/valitron/blob/v1.4.11/src/Valitron/Validator.php#L1251
```

the same for types

also sort them

Reply:
It doesn't work on same line because then it treats that URL as part of return type.
See https://phpstan.org/r/e6afc3cb-ad77-43be-9b74-933acc4bccb2 for example

Reply:
https://github.com/phpstan/phpstan/issues/10874